runtimes/common/src/constants.rs

```diff
@@ -303,6 +303,15 @@ pub mod governance {
 	}
 }
 
+pub mod multisig {
+	use super::*;
+	parameter_types! {
+		pub const MaxSignitors: u32 = 64;
+		pub const DepositBase: Balance = 50 * MILLI_KILT;
+		pub const DepositFactor: Balance = 10 * MILLI_KILT;
```

Comment:
Where do these numbers come from?

In theory we use the `deposit` function where ever a deposit is required. There we use the formula `items * 56 * MILLI_KILT + bytes * 50 * MICRO_KILT`. So I guess this should be:

```suggestion
		pub const MaxSignitors: u32 = 64;
		pub const DepositBase: Balance = 56 * MILLI_KILT;
		pub const DepositFactor: Balance = 50 * MILLI_KILT;
```

Reply:
BTW since we use these magic values now at two places could we create two constants for 50 and 56?

Reply:
Hope it's declarative enough: [192d07d](https://github.com/KILTprotocol/kilt-node/pull/498/commits/192d07de7aae0237c37f803f93e54833de2873b9)